tests/api_tests/collections/test_views.py

```diff
@@ -0,0 +1,716 @@
+# -*- coding: utf-8 -*-
+import mock
+from nose.tools import *  # flake8: noqa
+
+from framework.auth.core import Auth
+from website.models import Node
+from website.util.sanitize import strip_html
+
+from tests.base import ApiTestCase, fake
+from tests.factories import UserFactory, ProjectFactory, FolderFactory, DashboardFactory, NodeFactory
+
+class TestCollectionsList(ApiTestCase):
+    def setUp(self):
+        ApiTestCase.setUp(self)
+        self.user = UserFactory.build()
+        self.user.set_password('justapoorboy')
+        self.user.save()
+        self.basic_auth = (self.user.username, 'justapoorboy')
+
+        self.non_contrib = UserFactory.build()
+        self.non_contrib.set_password('justapoorboy')
+        self.non_contrib.save()
+        self.basic_non_contrib_auth = (self.non_contrib.username, 'justapoorboy')
+
+        self.deleted = ProjectFactory(is_deleted=True, is_folder=True)
+        self.private = ProjectFactory(is_public=False, creator=self.user, is_folder=True)
+        self.public = ProjectFactory(is_public=True, creator=self.user, is_folder=True)
+        self.url = '/v2/collections/'
+
+    def test_only_returns_non_deleted_public_projects(self):
+        res = self.app.get(self.url)
+        node_json = res.json['data']
+
+        ids = [each['id'] for each in node_json]
```

Comment:
Minor, but I prefer to use descriptive nouns instead of each. Since it's a comprehension, each is implied. Something like `[node['id'] for node in node_json]` will give better context for people reading through.